docs/spec/spec.md

```diff
@@ -0,0 +1,564 @@
+# Specification: Ballerina MQTT Library
+
+_Owners_: @shafreenAnfar @dilanSachi    
+_Reviewers_: @shafreenAnfar  
+_Created_: 2023/08/15    
+_Updated_: 2023/08/15   
+_Edition_: Swan Lake
```

Comment:
Most IDEs are configured to remove trailing spaces (If your IDE hasn't configured to do so, I'd advise you to ;)). In that case, these spaces will be removed automatically without us knowing. So the better approach would be to use the proper MD format here.

```suggestion
_Owners_: @shafreenAnfar @dilanSachi \
_Reviewers_: @shafreenAnfar \
_Created_: 2023/08/15 \
_Updated_: 2023/08/15 \  
_Edition_: Swan Lake
```

Reply:
Fixed